install.sh

```diff
@@ -70,6 +70,21 @@ if [ "${EUID}" == "0" ]; then
   exit 1;
 fi
 
+case "${ARCH}" in
+"i686")
+  echo_info 'The support for legacy 32-bit userspace (i686) is dropped on 2022-05-10,'
+  echo_info 'Consider switching to a 64-bit Chromium OS.'
+  exit 1
```

Comment:
Would a warning be better?  IOW, no `exit 1`.  Also, since arm Chromebooks still operate in a 32-bit userspace, maybe it would be better to say:
```
Support for i686 architecture has been dropped as of May 10, 2022.
```
?